### ISSUE TITLE
Disable certain controls until something is drawn

### ISSUE THREAD

**Issue by fedarko** [marbl/MetagenomeScope#266]
This was the case in old versions of MetagenomeScope -- as I remember, we disabled things (1) at the start, (2) during the process of loading a DB file, and (3) during the process of drawing something.

In recent (post-2020) versions of MetagenomeScope we have less things to worry about, since we are not swapping out entire graphs within the same visualization instance. However, it is still a good idea to disable some irrelevant controls before anything is drawn -- and during long drawing operations, too, I guess. (And then there is the attendant concern of avoiding flashing between enabled/disabled for quick operations... I put a lot of work into using transitions in old versions of MetagenomeScope to smooth this out. Although maybe the more elegant way to handle during-drawing stuff -- rather than disabling things -- is to just set some sort of `dcc.Store` with info about if we are currently drawing something, and checking at each callback if this is true before performing one operation during another. idk.)

Anyway, some things that it would be good to disable / etc. at certain points (just going by what is currently in the application, list will expand with time):

- fit button (should be disabled until the graph is drawn)
- draw button (i guess disable this while drawing something? looks like spamming the draw button slows down firefox, although maybe this is an issue with doing the layout in dagre. idk.)
- node / edge colorization checkboxes? maybe? or maybe we can just leave them be.
- the Cytoscape.js instance itself -- since it is persistent (and doesn't just get destroyed/created on redraws), you can still pan around the graph before anything is drawn.

This is all fairly low-priority -- mostly just small UX things and preventing people from breaking the display antagonistically.
